packages/fluxible-reducer-store/README.md

```diff
@@ -0,0 +1,118 @@
+# Fluxible Reducer Store
```

Comment:
the rest of this readme is wrong isn't it? if its package specific, it looks like a copy of the fluxible one.